src/main.rs

```diff
@@ -518,17 +562,22 @@ impl AsyncComponent for App {
         model.menu_popover = widgets.menu_popover.clone();
 
         if init.minimize_on_start {
-            root.minimize();
+            root.hide();
```

Comment:
my idea is that if the goal is to minimize, it would be better to directly hide it in the system tray. If you think minimizing is better, I can change it back here.

Reply:
That is not what I meant. You do not check if tray icon was enabled successfully. So if tray icon was not enabled correctly application will crash, but I think it shouldn't because tray icon is not a critical functionality. Instead tray icon should probably be stored in `Option<TrayIcon>` and we simply should minimize window instead of hiding in that case. Because if you hide it, you'll not be able to show it again without tray icon.